cpp/ql/test/library-tests/dataflow/taint-tests/atl.cpp

```diff
@@ -0,0 +1,899 @@
+namespace {
+  template<typename T> T source();
+  template<typename T> T* indirect_source();
+  void sink(...);
+}
+
+typedef unsigned int UINT;
+typedef long LONG;
+typedef void* LPVOID;
+typedef void* PVOID;
+typedef bool BOOL;
+typedef char* PSTR, *LPSTR;
+typedef const char* LPCTSTR;
+typedef unsigned short WORD;
+typedef unsigned long DWORD;
+typedef void* HANDLE;
+typedef LONG HRESULT;
+typedef unsigned long ULONG;
+typedef const char* LPCSTR;
+typedef wchar_t OLECHAR;
+typedef OLECHAR* LPOLESTR;
+typedef const LPOLESTR LPCOLESTR;
+typedef OLECHAR* BSTR;
+typedef wchar_t* LPWSTR, *PWSTR;
+typedef BSTR* LPBSTR;
+typedef unsigned short USHORT;
+typedef char *LPTSTR;
+struct __POSITION { int unused; };typedef __POSITION* POSITION;
```

Comment:
Can we have a newline between thee two declarations?
```suggestion
struct __POSITION { int unused; };
typedef __POSITION* POSITION;
```

Reply:
Done. Also updated the test changes that happened as a result of this: 593e2233f827f63161c664be9c54ec556c59ddde